webpack.config.js

```diff
@@ -0,0 +1,229 @@
+const path = require("path");
+const webpack = require("webpack");
+const CopyPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+const TerserJSPlugin = require("terser-webpack-plugin");
+
+// `module.exports` is NodeJS's way of exporting code from a file,
```

Comment:
Note: the reason this file has a crap ton of comments is so that folks who want to learn or don't know Webpack can learn from it.

Reply:
Comments are good :+1: